app/Http/Controllers/Solo/ScoresController.php

```diff
@@ -69,6 +69,7 @@ public function update($beatmapId, $id)
         $params['mods'] = Mod::parseInputArray($params['mods'] ?? [], $score->ruleset_id);
 
         $score->complete($params);
+        $score->createLegacyEntry();
```

Comment:
transaction? (because can't `complete` a second time if something errors in the next line)